lib/metadataTypes/MetadataType.js

```diff
@@ -203,7 +203,7 @@ class MetadataType {
      * @returns {Promise<{metadata:Util.MetadataTypeMap,type:string}>} metadata
      */
     static retrieveChangelog(additionalFields, buObject, subType) {
```

Comment:
only partially relevant - but if additionalFields is optional, then it should come last. Is there a reason for this?

Reply:
actually that reveiled / incorrectly passed variables. fixed that in my coming push